scripts/encrypt/index.ts

```diff
@@ -0,0 +1,33 @@
+import { encrypt } from "@opensystemslab/planx-core";
+
+/**
+ * Encrypt a secret
+ * Currently used to generate secure secrets for the team_integrations table
+ * e.g converting plain text 3rd-part API keys (such as BOPS tokens) to encrypted strings
+ *
+ * @param secret - The secret to be encrypted.
+ * @param encryptionKey - The encryption key - a 32-byte string
+ * @returns The encrypted secret and initialization vector in the format ${secret}:${iv}
+ * @example pnpm encode <secret> <encryptionKey> 
+ */
+function main() {
+  try {
+    if (process.argv.length < 4) {
+      console.error("Usage: pnpm encode <secret> <encryptionKey>");
```

Comment:
Is there any risk to logging out the `encryptionKey` / we generally avoid this with other env vars ? AWS logs are securely behind SSO I guess?

Reply:
This is actually just a straight up string, no templating so there's no real risk here I think.

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/f7c379f2-1505-4796-9962-1035e5e9cf91)

Reply:
aha of course - thanks for clarifying - I read as a template out of habit !